Library/Formula/sourcekitten.rb

```diff
@@ -0,0 +1,16 @@
+class Sourcekitten < Formula
+  homepage "https://github.com/jpsim/SourceKitten"
+  url "https://github.com/jpsim/SourceKitten.git", :tag => "0.3.0"
+  head "https://github.com/jpsim/SourceKitten.git"
+
+  depends_on :xcode => ["6.1.1", :build]
+
+  def install
+    system "make", "prefix_install", "PREFIX=#{prefix}"
+    system "make", "clean" # remove temporary installation files in /tmp/SourceKitten.dst
```

Comment:
I'd rather that Carthage didn't do that either, honestly. I don't think OS X does automatically clean that directory, no. We clean our `buildpath` automatically, though.